lib/viewing/ViewRelationship.js

```diff
@@ -112,13 +114,45 @@ ViewRelationship.prototype = {
     },
 
     renderChildViewIntoParent: function() {
-        var $destination = this.parentView.$el;
+        var destinationEl = this.parentView.el;
 
         if (this.destination) {
-            $destination = this.parentView.$(this.destination);
+            destinationEl = chooseDestination(destinationEl, this.destination);
         }
 
-        $destination[this.placementStrategy](this.childView.render().el);
+        if (destinationEl) {
+            var childViewEl = this.childView.render().el;
+
+            switch (this.placementStrategy) {
+                case "append":
+                    destinationEl.appendChild(childViewEl);
+                    break;
+
+                case "prepend":
+                    destinationEl.insertBefore(childViewEl, destinationEl.firstChild);
+                    break;
+
+                case "html":
+                    while (destinationEl.firstChild) {
+                        destinationEl.removeChild(destinationEl.firstChild);
+                    }
+
+                    destinationEl.appendChild(childViewEl);
+                    break;
+
+                case "before":
+                    destinationEl.parentNode.insertBefore(childViewEl, destinationEl);
+                    break;
+
+                case "after":
+                    destinationEl.parentNode.insertBefore(childViewEl, destinationEl.nextSibling);
+                    break;
+
+                case "replaceWith":
+                    destinationEl.parentNode.replaceChild(childViewEl, destinationEl);
+                    break;
+            }
```

Comment:
might be a good ideal to add a `default` here.

Reply:
true story. I wasn't sure what the default should be though? what do you think?

Reply:
Might it be reasonable to throw with a nice error message?

Reply:
ah great idea, will do